packages/solarwinds-apm/package.json

```diff
@@ -45,8 +45,11 @@
     "test": "swtest -p test/tsconfig.json -c src"
   },
   "dependencies": {
+    "@grpc/grpc-js": "^1.9.3",
```

Comment:
Is this for later in our solarwinds-apm package we have to muck with `grpc.Metadata()`? And I guess somehow the OTLP grpc exporter packages don't have this as a dependency... or they do but it's hard for us to reference?